static/style.css

```diff
@@ -1,31 +1,75 @@
 body {
-  background-color: lightblue;
+  background-color: FEFEFE;
   text-align: center;
-  font-family: Arial, Helvetica, sans-serif;
-}
-
-img {
-  border-radius: 25px;
+  margin-top: 100px;
 }
 
 audio {
   padding-bottom: 30px;
 }
 
+
+a {
+  color: inherit;
+}
+
+.btn {
+  display: inline-block;
+}
+
 .feedback {
   margin: 0 auto;
 }
 
-.container{
+.container {
   margin: auto;
   width: 50%;
   border: 3px springgreen;
   padding: 10px;
-  
-  
 }
-.background{
-  color:rgb(6, 107, 85) ;
+
+.background {
+  color: rgb(6, 107, 85);
 }
 
+.bgimg {
```

Comment:
This is the old bgimg that I replaced with the faded version that covers the whole screen in line 45.